classes/Carrier.php

```diff
@@ -520,16 +520,16 @@ public static function getCarriers($id_lang, $active = false, $delete = false, $
         }
 
         switch ($modules_filters) {
-            case 1 :
```

Comment:
The installer was broken because it was trying to use `array_map()`on `null`(https://github.com/PrestaShop/PrestaShop/pull/6377/files#diff-6aa7ebc128f73c6ad0299c4fddf4c562R1564)